docs/_templates/2018/index.html

```diff
@@ -91,11 +91,22 @@
 
   <div class="about">
     <div class="uk-container">
+
       <div class="about-content">
         <h2>What is Write the Docs about?</h2>
         <p>Write the Docs brings everyone who writes the docs together in the same room: Programmers, Tech Writers, Support, Designers, Developer Advocates, and more. We all have things to learn from each other, and there’s no better way than sitting together and talking.</p>
         <p>We invite you to join {{ about.attendees }} other folks in our event to explore the art and science of documentation.</p>
+
+        {% if cobrand.logo %}
+        <h2>What is Open Help about?</h2>
```

Comment:
would these headings, inc WTD, be better without "about"?

Reply:
I agree -- "What is Write the Docs?" and "What is Open Help?" sound better to me.